docs-markdown/src/controllers/cleanup/utilities.ts

```diff
@@ -8,12 +8,18 @@ const recursive = require("recursive-readdir");
 let percentComplete = 1;
 /**
  * Checks if array has only one item. if so, then return that item.
- * @param content takes in string data as content and retuns
+ * @param content takes in string data as content and returns
  * first item in array if the array only has one item in array.
  */
 export function handleSingleItemArray(content: string | undefined) {
-    if (content && Array.isArray(content) && content.length === 1) {
-        return content[0];
+    if (content && content.length === 1) {
+        if (content[0] === "") {
+            // It does not matter what we return
+            return " ";
```

Comment:
if the metadata attribute is empty, we probably dont want to add a space? I'd rather just remove the brackets if its an empty array.

Reply:
In my testing, it does not matter what value is returned as long as it is not empty or null. I'm returning a space here as a simple value. We appear to be able to use:
```typescript
export function handleSingleItemArray(content: string | undefined) {
    if (content && content.length === 1) {
        if (content[0] === "") {
            return true;
        }
        else {
            return content[0];
        }
    }
}
```
However, this then suggests that this function should be renamed to `isSingleItemArray`. Please let me know your thoughts on these changes.

Reply:
sounds good to me. Id prefer `isSingleItemArray`